sonar-db/src/main/java/org/sonar/db/permission/GroupPermissionMapper.java

```diff
@@ -34,23 +34,11 @@
   @Deprecated
   int countGroups(Map<String, Object> parameters);
 
-  /**
-   * @deprecated does not support organizations
-   */
-  @Deprecated
-  List<String> selectGroupNamesByPermissionQuery(@Param("query") PermissionQuery query, RowBounds rowBounds);
+  List<Long> selectGroupIdsByQuery(@Param("organizationUuid") String organizationUuid, @Param("query") PermissionQuery query, RowBounds rowBounds);
 
-  /**
-   * @deprecated does not support organizations
-   */
-  @Deprecated
-  int countGroupsByPermissionQuery(@Param("query") PermissionQuery query);
+  int countGroupsByPermissionQuery(@Param("organizationUuid") String organizationUuid, @Param("query") PermissionQuery query);
 
-  /**
-   * @deprecated does not support organizations
-   */
-  @Deprecated
-  List<GroupPermissionDto> selectGroupPermissionByGroupNames(@Param("groupNames") List<String> groupNames, @Nullable @Param("projectId") Long projectId);
+  List<GroupPermissionDto> selectByGroupIds(@Param("organizationUuid") String organizationUuid, @Param("groupIds") List<Long> groupIds, @Nullable @Param("projectId") Long projectId);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 182 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS00103)